Resources/Prototypes/Entities/Mobs/Customization/Markings/tattoos.yml

```diff
@@ -72,7 +72,7 @@
   id: TattooCampbellRightArm
   bodyPart: RArm
   markingCategory: Arms
-  speciesRestriction: [Human, Dwarf, Felinid, Oni] # Delta V - Felinid, Oni
```

Comment:
And here

Reply:
Looks like more cases where this PR previously conflicted with another PR that added certain markings to more species. It conflicted with this PR at the time since it was allowing Harpies to use markings that in this PR would conflict with their new visuals. I've submitted a fix for these conflicts.